### ISSUE TITLE
Corrupted output

### ISSUE THREAD

**Issue by enotanet** [SPQRInc/pagekit-glossary#1]
In case of therm in glossary start not from new line rise corrupted output html:

Input example: test cryptography test1. 
Dump of SQL to reproduce error:

> insert  into `pk1011_glossary_item`(`id`,`slug`,`status`,`title`,`marker`,`content`,`excerpt`,`data`,`date`,`modified`) values (2,'aes',1,'AES','[\"aes\",\"Advanced Encryption Standard\"]','The Advanced Encryption Standard (AES), also known by its original name **Rijndael** (Dutch pronunciation: Л€rЙ›indaЛђl, is a specification for the encryption of electronic data established by the U.S. National Institute of Standards and Technology (NIST) in 2001.\n\nAES is a subset of the Rijndael cipher **developed by two Belgian cryptographers**, **Vincent Rijmen** and **Joan Daemen**, who submitted a proposal to NIST during the AES selection process. Rijndael is a family of ciphers with different key and block sizes.\n\nFor AES, NIST selected three members of the Rijndael family, each with a block size of 128 bits, but three different key lengths: 128, 192 and 256 bits.\n\nAES has been adopted by the U.S. government and is **now used worldwide**. It supersedes the Data Encryption Standard (DES), which was published in 1977. The algorithm described by **AES is a symmetric-key algorithm**, meaning the same key is used for both encrypting and decrypting the data.','Advanced Encryption Standard. The first (and only) publicly accessible cipher approved by the National Security Agency (NSA) for top secret information when used in an NSA approved cryptographic module.','{\"markdown\":true}','2017-06-03 13:15:00','2017-06-03 13:39:04'),(3,'cryptanalysis',1,'Cryptanalysis','[]','The study of methods for obtaining the meaning of encrypted information, without access to the secret information that is normally required to do so.','','{\"markdown\":true}','2017-06-03 13:18:36','2017-06-03 13:18:59'),(4,'cryptography',1,' Cryptography','[\"cryptography\",\"cryptology\"]','Cryptography or cryptology (from Greek ОєПЃП…ПЂП„ПЊП‚ kryptГіs, \"hidden, secret\"; and ОіПЃО¬П†ОµО№ОЅ graphein, \"writing\", or -О»ОїОіОЇО± -logia, \"study\", respectively) is the practice and study of techniques for secure communication in the presence of third parties called adversaries. More generally, cryptography is about constructing and analyzing protocols that prevent third parties or the public from reading private messages; various aspects in information security such as data confidentiality, data integrity, authentication, and non-repudiation are central to modern cryptography. Modern cryptography exists at the intersection of the disciplines of mathematics, computer science, and electrical engineering. Applications of cryptography include military communications, electronic commerce, ATM cards, and computer passwords.','Common expression to refer to process of encrypting and decrypting information.','{\"markdown\":true}','2017-06-03 13:19:31','2017-06-03 16:47:19');

**Comment by enotanet** [comment]
I found that first symbol in title SPACE breaks output.

**Comment by enotanet** [comment]
```
To fix I used simple trim 'title' before save.

public function saveAction( $data, $id = 0 )
	{
+		$data[ 'title' ] = trim($data[ 'title' ]);

		if ( !$id || !$item = Item::find( $id ) ) {
			if ( $id ) {
				App::abort( 404, __( 'Item not found.' ) );
			}
			$item = Item::create();
		}
		if ( !$data[ 'slug' ] = App::filter( $data[ 'slug' ] ? : $data[ 'title' ], 'slugify' ) ) {
			App::abort( 400, __( 'Invalid slug.' ) );
		}
		
		$item->save( $data );
		
		return [ 'message' => 'success', 'item' => $item ];
	}

```

**Comment by enotanet** [comment]
Also I found that any of "()[]{}" in excerpt gives corrupted output. I did not fix it.

**Comment by ahoiroman** [comment]
I did. Please wait for the next release.

**Comment by ahoiroman** [comment]
Could you please provide more information? Which Pagekit-version are you using? Which database-backend?

Using Pagekit 1.0.11 with SQLite I can not reproduce this error.

**Comment by enotanet** [comment]
I can provide you full access to my web-site.
I did not find contact on your web-site.

Pagekit Version 	1.0.11
Web Server 	Apache/2.4.18 (Ubuntu)
User Agent 	Mozilla/5.0 (Windows NT 5.1; rv:50.0) Gecko/20100101 Firefox/50.0
PHP Version 	7.0.18-0ubuntu0.16.04.1
Driver 	mysql
Version 	5.7.18-0ubuntu0.16.04.1

**Comment by ahoiroman** [comment]
You can contact me at roman@spqr.wrf or just using gitter or on pagekit-forum.org

**Comment by enotanet** [comment]
I have sent email a few days ago. What can I do to help you collect maximum info you may need?

**Comment by ahoiroman** [comment]
Please be a little patient ;) There have been public holidays in Germany and writing this extension is not my food-job ;) I will answer as soon as I got time for this.

**Comment by ahoiroman** [comment]
Did not receive any message yet. But there is a really picky spam-protection working. I disabled it for now, maybe you can resend the mail?

**Comment by enotanet** [comment]
O god, work done like this is your main paid project )

**Comment by enotanet** [comment]
I sent message again to: roman@spqr.wtf

mail you provided is not available: roman@spqr.wRf

**Comment by ahoiroman** [comment]
Oh sorry, it was autocorrect (just wondering, why it is correcting to wRf).

Did not receive your mail yet, but I will wait a few minutes.

**Comment by enotanet** [comment]
hm.. status delivered..
now I am going to reply from google account..

**Comment by enotanet** [comment]
just wondering, why it is correcting to wRf

may be auto speller predicted bad word? ))

**Comment by ahoiroman** [comment]
Closed with coming release `1.0.5`.
